packages/react-composites/src/composites/common/TabHeader.tsx

```diff
@@ -79,6 +79,11 @@ export const PeopleAndChatHeader = (props: PeopleAndChatHeaderProps): JSX.Elemen
             onClick={onChatButtonClicked}
             styles={mobilePaneButtonStylesThemed}
             checked={activeTab === 'chat'}
+            ariaLabel={
+              activeTab === 'chat'
+                ? strings.sidePaneActiveTabAriaLabel.replace('{header}', strings.chatButtonLabel)
```

Comment:
Out of interest, would setting [aria-selected](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected) property do the same thing without updating the string?

Reply:
That's a great suggestion!
Let me try this and report back

Reply:
It worked! Thanks James, this is a better cleaner fix.